### PR TITLE
add: Rustlings (rust)

### LINKED ISSUE #1788: [Add] Add rustlings to Rust
### Site URL

https://rustlings.cool/

### Description of the above provided site

This project contains small exercises to get you used to reading and writing Rust code.

### Category

Languages

### Sub Category

Rust

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by bencinn**
## Fixes Issue
Closes #1788 

## Changes proposed
Add rustlings to the Rust category

## Screenshots
image

## Note to reviewers
Hacktoberfest

### CODE REVIEW

File: database/languages/rust.json [hunk 1]
```diff
@@ -19,5 +19,12 @@
         "url": "https://www.youtube.com/watch?v=XZtlD_m59sM",
         "category": "languages",
         "subcategory": "rust"
+    },
+    {
+      "name": "Rustlings",
+      "description": "Rustlings is a collection of small exercises that get you used to reading and writing Rust code. This includes reading and responding to compiler messages!",
```
Comment (CBID2) [inline database/languages/rust.json:25]:
```suggestion
      "description": "Rustlings is a collection of small exercises that get you used to readin and writing in Rust. This includes reading and responding to compiler messages!",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @bencinn! :) I pointed out a  proofreading error that could be improved. You'll see it in green. From there, just click on "Commit Suggestion", rewrite the commit message with a more conventional one (e.g., `fix: message here`), and press "Commit Changes". Other than that, you're off to a good start!! :)

**Comment by bencinn** [comment]
> Hi @bencinn! :) I pointed out a proofreading error that could be improved. You'll see it in green. From there, just click on "Commit Suggestion", rewrite the commit message with a more conventional one (e.g., `fix: message here`), and press "Commit Changes". Other than that, you're off to a good start!! :)

Fixed :)

**Review by CBID2 — APPROVED** [review]
Well done @bencinn! :) Now you just need a second approved check :)

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by CBID2** [comment]
Yay! You did it @bencinn! :)

**Pull request merged**